tomviz/tomvizPythonConfig.h.in

```diff
@@ -83,6 +83,8 @@ void PythonAppInitPrependPathLinux(const std::string& exe_dir)
 
     // Add tomviz Python source dir.
     PythonAppInitPrependPythonPath(TOMVIZ_PYTHON_SOURCE_DIR);
+
+    PythonAppInitPrependPythonPath("/home/patrick/virtualenvs/tomviz/lib/python3.8/site-packages");
```

Comment:
Not sure we want this!

Reply:
Whoops. This was an accident. Good thing you are reviewing :wink: